db/config.cc

```diff
@@ -871,6 +871,9 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_sstables_mc_format(this, "enable_sstables_mc_format", value_status::Unused, true, "Enable SSTables 'mc' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , enable_sstables_md_format(this, "enable_sstables_md_format", value_status::Unused, true, "Enable SSTables 'md' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , sstable_format(this, "sstable_format", value_status::Used, "me", "Default sstable file format", {"md", "me"})
+    , uuid_sstable_identifiers_enabled(this,
```

Comment:
other config values that enable stuff start with the `enable_` prefix, instead of ending with the `_enabled` suffix.

Reply:
see https://github.com/scylladb/scylladb/pull/13073#discussion_r1136593172 . we had a similar discussion back then.

Reply:
Don't think we care about compatibility on the config option naming level.

Reply:
It's better not to be gratuitously incompatible, it will just cause surprises.

Reply:
great. will use the more consistent name for the variable and option in the next revision.

Reply:
Wait, what's the Cassandra name?

Reply:
> Wait

https://github.com/apache/cassandra/blob/edd35badaf6e25726b0e46a4cddf9cfe42f0c11a/conf/cassandra.yaml#L1037

Reply:
So let's use that.

Reply:
okay. i will keep using the name of "uuid_sstable_identifiers_enabled" for the option and for the variable. even it is not consistent with other options/variables.